### PR TITLE
Updating references to 12 word seed phrases in copy

### LINKED ISSUE #10604: Correct wording on seed phrase import screen
Screen Shot 2021-03-08 at 10 00 37 AM

We currently support various lengthed seed phrase imports, but we still say "twelve word phrase" in some of the copy.

### PR DESCRIPTION
**Pull request by ryanml**
Fixes: https://github.com/MetaMask/metamask-extension/issues/10604

**Screens with updated copy**
Screen Shot 2021-03-23 at 5 59 17 PM

Screen Shot 2021-03-23 at 5 59 26 PM

**Padding fix**

Before
Screen Shot 2021-03-23 at 6 08 56 PM

After
Screen Shot 2021-03-23 at 6 09 15 PM

### CODE REVIEW

File: ui/app/pages/keychains/index.scss [hunk 1]
```diff
@@ -25,6 +25,12 @@
   margin: 60px 0 30px 0;
   position: relative;
   max-width: initial;
+
+  &__input-label {
```
Comment (ryanml) [inline ui/app/pages/keychains/index.scss:29]: There was no rule defined for `import-account__input-label` so I think this rule just was forgotten to be added when this component was made (in line with other text label styling)

**Comment by ryanml** [comment]
cc: @danfinlay just to confirm updated copy looks fine

File: app/_locales/en/messages.json [hunk 2]
```diff
@@ -1455,7 +1455,7 @@
     "message": "WARNING: Never disclose your backup phrase. Anyone with this phrase can take your Ether forever."
   },
   "secretPhrase": {
-    "message": "Enter your secret twelve word phrase here to restore your vault."
+    "message": "Enter your secret phrase here to restore your vault."
```
Comment (brad-decker) [inline app/_locales/en/messages.json:1458]: With this change (and `importYourExisting`) we'll want to make sure our other language files get updated to reflect the difference as well. I still don't know what's the best practice here, if we can just resubmit this file for translation of the keys that have been updated or what.

**Review by brad-decker — DISMISSED** [review]
LGTM other than that translation issue. Lets bring this up in the extension sync today.

File: app/_locales/en/messages.json (hunk 2 above)
Comment (Gudahtt) [reply app/_locales/en/messages.json:1458]: The best approach we have is to remove the translations and hope somebody re-adds them. Not a great process, but, hopefully we'll have a better one soon for some locales at least.

Comment (ryanml) [reply app/_locales/en/messages.json:1458]: As discussed on sync, taking the approach @Gudahtt mentioned for now, translations updated

**Review by Gudahtt — APPROVED** [review]
LGTM!

**Review by brad-decker — APPROVED** [review]
LGTM

**Pull request merged**